staging/src/k8s.io/apiserver/pkg/util/flag/flags.go

```diff
@@ -43,12 +45,16 @@ func WarnWordSepNormalizeFunc(f *pflag.FlagSet, name string) pflag.NormalizedNam
 	return pflag.NormalizedName(name)
 }
 
-// InitFlags normalizes, parses, then logs the command line flags
+// InitFlags normalizes, parses the command line flags
 func InitFlags() {
 	pflag.CommandLine.SetNormalizeFunc(WordSepNormalizeFunc)
 	pflag.CommandLine.AddGoFlagSet(goflag.CommandLine)
 	pflag.Parse()
-	pflag.VisitAll(func(flag *pflag.Flag) {
-		glog.V(2).Infof("FLAG: --%s=%q", flag.Name, flag.Value)
-	})
 }
+
+// PrintFlags logs the command line parameters
+func PrintFlags() {
+	for _, arg := range os.Args {
+		log.Printf("FLAG: %v", arg)
```

Comment:
why not glog.V(2).Infof?

Reply:
also, os.Args is not the same as flags... does this mean `kubelet -h` would print this?
```
FLAG: kubelet
FLAG: -h
```

Reply:
`log.Printf` was from trial and error. `glog.V(2).Infof` does not work at this point.

let me check on the 'kubectl -h`

Reply:
hmm, you meant kubelet not kubectl

Reply:
yes, sorry